docs/source/models/lasers.rst

```diff
@@ -0,0 +1,238 @@
+.. _model-lasers:
+
+Analytic Expressions for the 3D Laser Profiles
+==============================================
+
+.. sectionauthor:: Klaus Steiniger
+
+
+Among others, PIConGPU offers the ``GaussianPulse`` and ``DispersivePulse`` profiles to model complex, Gaussian-like laser pulse profiles in simulations.
+
+The ``GaussianPulse`` profile allows modelling standard Gaussian pulses defined by a Gaussian distribution of laser energy along the longitudinal and transverse directions.
+Furthermore, it allows including higher-order transverse modes via Laguerre-Gaussian modes [Pausch2022]_.
+
+The ``DispersivePulse`` profile allows modelling standard Gaussian pulses which feature dispersions, i.e. frequency specific modifications of the laser's spectral phase, up to third order.
+That is, time delay (TD), angular dispersion (AD), spatial dispersion (AD), group-delay dispersion (GDD), and third-order dispersion (TOD) are self-consistently taken into account in the propagation of these pulses.
+The profile assumes a gaussian shape for the laser's spectrum.
+The respective in-focus values of these dispersions can be provided as parameters.
+However, the electric field values in time domain are computed from the field's values in frequency domain by a discrete Fourier transform.
+Therefore, it is possible to use any other shape of the laser's spectrum by modifying the profile's source code.
+
+A very concise description of the equations used in the ``GaussianPulse`` and ``DispersivePulse`` profiles are provided in the following.
+Please also refer to the in-code documentation of these and the other profiles in order to obtain more profile-specific information.
+Furthermore, refer to :ref:`Workflows / Adding Laser <usage-workflows-addLaser>` in order to see how to add a laser to a PIConGPU simulation.
+
+
+
+Definitions
+-----------
+Following [Steiniger2024]_, the electric field in frequency-space domain is assumed to be polarized along :math:`x` and defined as
+
+.. math::
+  \hat{\vec E}(\vec r, \Omega) = \hat E_\mathrm{A}(\vec r, \Omega) e^{-\imath \varphi(\vec r, \Omega)}\vec{\mathrm e}_x\,,
+
+where :math:`\Omega=2\pi\nu` is the angular frequency and :math:`\vec r` the position considered, :math:`\hat E_\mathrm{A}` is the spectral amplitude and
+:math:`\varphi=\tfrac{\Omega}{c} \vec{\mathrm e}_\Omega \cdot \vec r`
+the spectral phase of the pulse,
+with :math:`\vec{\mathrm e}_\Omega` being the propagation direction of frequency :math:`\Omega`.
+Dispersions are assumed to occur only in the plane spanned by the direction of pulse propagation :math:`\vec{\mathrm e}_z`, being equal to the direction of propagation of the central frequency :math:`\Omega_0`, and polarization :math:`\vec{\mathrm e}_x`, i.e. :math:`\vec{\mathrm e}_\Omega \cdot \vec{\mathrm e}_y = 0`.
+This implies, that the spectral phase does not vary along the :math:`y`-direction in focus, i.e.
```

Comment:
```suggestion
This implies that the spectral phase does not vary along the :math:`y`-direction in focus, i.e.
```

Reply:
Okay -it's clear now: y-direction here is orthogonal to laser polarization in x and propagation in z. 
Fine.